resources/views/autor.blade.php

```diff
@@ -31,7 +31,7 @@
     </section>
 @endif
 
-<section class="author detail content-section" itemscope itemtype="http://schema.org/Person">
+<section class="author detail content-section" itemscope itemtype="http://schema.org/{{ $author->type == 'corporate body' ? 'Organization' : 'Person' }}">
```

Comment:
I can see these checks for "is corporate body?" a bit. Maybe a helper like `$author->is_corporate_body` (or `is_person`) would be useful?
```suggestion
<section class="author detail content-section" itemscope itemtype="http://schema.org/{{ $author->is_corporate_body ? 'Organization' : 'Person' }}">
```